src/pyOpenMS/pxds/LPWrapper.pxd

```diff
@@ -7,48 +7,74 @@ cdef extern from "<OpenMS/DATASTRUCTURES/LPWrapper.h>" namespace "OpenMS":
+        double getColumnLowerBound(Int index) nogil except + # wrap-doc:Gets column's lower bound
+        double getRowUpperBound(Int index) nogil except + # wrap-doc:Gets row's upper bound
+        double getRowLowerBound(Int index) nogil except + # wrap-doc:Gets row's lower bound
+        void setRowName(Int index, const String & name) nogil except + # wrap-doc:Sets name of the index-th row
+        void setColumnBounds(Int index, double lower_bound, double upper_bound, LPWrapper_Type type_) nogil except + # wrap-doc:Set column bounds
+        void setRowBounds(Int index, double lower_bound, double upper_bound, LPWrapper_Type type_) nogil except + # wrap-doc:Set row bounds
+        void setColumnType(Int index, VariableType type_) nogil except + # wrap-doc:Set column/variable type.
+        VariableType getColumnType(Int index) nogil except + # wrap-doc:Get column/variable type.
+        void setObjective(Int index, double obj_value) nogil except + # wrap-doc:Set objective value for column with index
+        double getObjective(Int index) nogil except + # wrap-doc:Get objective value for column with index
+        void setObjectiveSense(Sense sense) nogil except + # wrap-doc:Set objective direction
         Sense getObjectiveSense() nogil except +
-        Int getNumberOfColumns() nogil except +
-        Int getNumberOfRows() nogil except +
-        void setElement(Int row_index, Int column_index, double value) nogil except +
-        double getElement(Int row_index, Int column_index) nogil except +
+        Int getNumberOfColumns() nogil except + # wrap-doc:Get number of columns
+        Int getNumberOfRows() nogil except + # wrap-doc:Get number of rows
+        void setElement(Int row_index, Int column_index, double value) nogil except +# TODO
+        double getElement(Int row_index, Int column_index) nogil except +# TODO
         void readProblem(String filename, String format_) nogil except +
+            # wrap-doc:
+                #   Read LP from file
+                #   -----
+                #   :param filename: Filename where to store the LP problem
+                #   :param format: LP, MPS or GLPK
+
         void writeProblem(const String & filename, WriteFormat format_) nogil except +
+            # wrap-doc:
+                #   Write LP formulation to a file
+                #   -----
+                #   :param filename: output filename, if the filename ends with '.gz' it will be compressed
+                #   :param format: MPS-format is supported by GLPK and COIN-OR; LP and GLPK-formats only by GLPK
+
         Int solve(SolverParam & solver_param, Size verbose_level) nogil except +
+            # wrap-doc:
+                #   solve problems, parameters like enabled heuristics can be given via solver_param
+                #   -----
+                #   The verbose level (0,1,2) determines if the solver prints status messages and internals.
+                #   -----
+                #   :param solver_param:
```

Comment:
Explanation missing (in the wrapper of this function)

Reply:
I didn't found the explanation in the header file. So left it blank.

Reply:
e.g. `Parameters of the solver introduced by SolverParam`

Reply:
Added